main/writer.c

```diff
@@ -27,6 +27,11 @@ static tagWriter *writerTable [WRITER_COUNT] = {
 
 static tagWriter *writer;
 
+extern void setCustomTagWriter(tagWriter *w)
```

Comment:
Currently this isn't very clean because we have to re-use some of the predefined parser types - the demo uses WRITER_U_CTAGS but there should be some support for custom writer type.